content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -23,21 +23,45 @@ Redis implements rolling updates for software upgrades in Kubernetes deployments
 
 ## Before upgrading
 
-Before you upgrade, verify you are using the correct image and your license is valid.
+Review the following warnings before starting your upgrade.
 
-{{< warning >}}
-When upgrading existing Redis Enterprise clusters running on RHEL7-based images, make sure to select a RHEL7-based image for the new version. See [release notes]({{<relref "/kubernetes/release-notes/">}}) for more info. 
-  {{</warning>}}
+#### OpenShift clusters running 6.2.12 or earlier
+
+Due to a change in the SCC, on OpenShift clusters running version 6.2.12 or earlier upgrading to version 6.2.18 or later, where `node:1` is <b>not</b> the master node, the upgrade might get stuck.
+
+You might have a pod that doesn't become fully ready and start seeing restarts. In the ServicesRigger log, you'll see this error:
+
+```sh
+services-rigger.services - ERROR - couldn't update pod <POD NAME> labels
+...
+... pods <POD NAME> is forbidden: unable to validate against any security context constraint
+```
+
+To prevent this, set `node:1` as the master node.
+
+```sh
+oc exec -it <rec-pod-name> -- rladmin cluster master set 1
+```
+
+This may also effect OpenShift clusters containing two RECs running different versions, if one is running version 6.2.12 or earlier and the other is running 6.2.18 or later.
```

Comment:
```suggestion
This may also affect OpenShift clusters containing two RECs running different versions, if one is running version 6.2.12 or earlier and the other is running 6.2.18 or later.
```